atmo/jobs/provisioners.py

```diff
@@ -16,6 +16,11 @@ class SparkJobProvisioner(Provisioner):
 
     def __init__(self):
         super().__init__()
+        # the S3 URI to the zeppelin setup step
+        self.zeppelin_uri = (
+            's3://%s/steps/zeppelin/zeppelin.sh' %
+            constance.config.AWS_SPARK_EMR_BUCKET
+        )
```

Comment:
Since this is in both provisioner subclasses we can move this to the base class.